app/models/chargeback_rate.rb

```diff
@@ -11,6 +20,8 @@ class ChargebackRate < ApplicationRecord
   validates_uniqueness_of   :guid
   validates_uniqueness_of   :description, :scope => :rate_type
 
+  virtual_column :assigned?, :type => :boolean
```

Comment:
Ah, I see.  This might work a bit better:

```ruby
 class User < ActiveRecord::Base
  before_destroy :nope

  def nope
    errors.add :id, "no destroying this!"
    throw :abort
  end

  def destroyable?
    run_callbacks :destroy
  end
end

user = User.create
if user.destroyable?
  puts "great!"
else
  puts "nope!"
end
```

The only thing that worries me about this trick is if the callbacks have side effects.  If that's the case, then it might just be better keep the `destroyable?` method and call the methods that know if it's destroyable.